packages/gatsby-theme-api-docs/src/components/api/resource/method/url-scopes-responses/scopes.js

```diff
@@ -1,22 +1,21 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import styled from '@emotion/styled';
+import { customProperties } from '@commercetools-uikit/design-system';
 import { Markdown } from '@commercetools-docs/ui-kit';
 
-const Title = styled.div`
-  color: #999;
+import Title from './title';
+
+const ListContainer = styled.p`
+  margin-top: ${customProperties.spacingXs};
 `;
 
 const Scopes = ({ scopes, title }) => {
   return (
     <div>
```

Comment:
You can use here SpacingsStack instead of defining a margin-top to the ListContainer